charts/kubescape-operator/templates/configs/cloud-secret.yaml

```diff
@@ -0,0 +1,12 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: {{ .Values.global.cloudSecret }}
+  namespace: {{ .Values.ksNamespace }}
+  labels:
+    app: {{ .Values.global.cloudSecret }}
+    tier: {{ .Values.global.namespaceTier }}
+type: Opaque
+data:
+  account-id: {{ .Values.account | default "" | b64enc }}
```

Comment:
```suggestion
  account: {{ .Values.account | default "" | b64enc }}
```

Reply:
I agree, we just need to make sure the CLI can load the data from the secret instead.